autoswitch_virtualenv.plugin.zsh

```diff
@@ -277,6 +277,21 @@ function _missing_error_message() {
     printf "\n"
```

Comment:
dont forget to remove the lines that check pwgen is installed!

Also, please update the README so that it no longer mentions pwgen as a requirement :)